plugins/wallabag.koplugin/main.lua

```diff
@@ -1003,7 +1017,18 @@ end
 
 function Wallabag:onAddWallabagArticle(article_url)
     if not NetworkMgr:isOnline() then
-        NetworkMgr:promptWifiOn()
+        UIManager:show(ConfirmBox:new{
+            text = _("No Wi-Fi connection. Connect now or add to queue for later download?"),
```

Comment:
If the proposed online/offline distinction is useful (of which I'm not convinced) then I'd suggest solving the problem elsewhere for better UX instead of constantly having to deal with a popup.

Reply:
What you propose?

Reply:
I'd just silently add it to the queue. Is there a particular reason not to?

Reply:
let me at least display for 1 second info that it has been added to queue. Ok?

Reply:
Yes, I just meant the logical alternative to what happens when online.